src/gengowatcher/ui_textual.py

```
@@ -969,8 +935,9 @@ def compose(self) -> ComposeResult:
                 with Vertical(id="dashboard-content"):
                     with Container(classes="dashboard-grid"):
                         yield JobsPreview(state=self.state)
+                        yield HourlyActivity(stats=self.stats)
+                        yield ConfigPreview()  # Keep as bottom-right per doc
                         yield JobsHourChart(stats=self.stats)
```

Ruff (0.14.14)

Undefined name `JobsHourChart`

(F821)